inc/osdconfig.h

```diff
@@ -7,10 +7,10 @@
 #define EERROM_SIZE                             1024
 
 // Version number: major.minor.revision (1.2.6 for example)
-#define PLAYUAV_VERSION_NUMBER          "1.1.2"
+#define PLAYUAV_VERSION_NUMBER          "1.1.3"
```

Comment:
according to semver it should be 1.2.0 (feature addition, not bug fix) not sure if we want to stick to it but after getting used to it I really appreciate knowing what type of changes were done